src/api-specification/contribution-controller/submitting-a-contribution-flag.md

```diff
@@ -0,0 +1,81 @@
+# Submitting a contribution flag
+
+> Submits [an assembled contribution flag](assembling-a-contribution-flag.md).
```

Comment:
should there be a reference to the general section with overview what the flag is?